jobdefs/jobtemplate.aurora

```diff
@@ -4,10 +4,15 @@ import os
 {{ ps.name }} = Process( name = '{{ ps.name }}', cmdline = '{{ ps.cmd }}' )
 {% endfor %}
 
+{% for ps in finalizers %}
+{{ ps.name }} = Process( name = '{{ ps.name }}', cmdline = '{{ ps.cmd }}', final=True )
+{% endfor %}
+
 {% for task in tasks %}
-{{ task.name }} = {{ task.type }}(
+{{ task.name }} = Task(
     name = '{{ task.name }}',
     processes = [ {{ task.processes|join(', ') }} ],
+    constraints = order( {{ task.ordering|join(', ') }} ),
```

Comment:
This makes up for the task no longer being implicitly ordered by SequentialTask; we specify the ordering of all non-finalizing processes (i.e. the commandline and input/output localization).